__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const request = require("supertest");
+const app = require("../app");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+describe("GET /api/categories", () => {
+  test("should return status 200 and an array of all categories", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((res) => {
+        const { body: categories } = res;
+        expect(Array.isArray(categories)).toBeTruthy();
+        categories.forEach((index) => {
+          expect(index).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
```

Comment:
I think youre missing a test case here, what might happen if I were to give you a path that didn't exist 🤔